### PR TITLE
added ability to configure meeting search radius if wanted

### PR DESCRIPTION
**Pull request by pjaudiomv**
this is a feature that i believe is needed for more dense regions and maybe smaller areas, not sure if should add to readme or not.

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -84,10 +84,13 @@ function helplineSearch($latitude, $longitude) {
 }
 
 function meetingSearch($meeting_results, $latitude, $longitude, $search_type, $day) {
+    if ($GLOBALS['meeting_search_radius'] == null) {
+        $GLOBALS['meeting_search_radius'] = -50;
+    }
```
Comment (radius314) [inline functions.php]: Use isset instead of checking for null.

**Comment by pjaudiomv** [comment]
wasnt sure how to update pull request after you requested changes and me commiting them

**Comment by radius314** [comment]
You just commit to your branch

File: README.md [hunk 2]
```diff
@@ -33,6 +33,21 @@ static $result_count_max = 10;
...
+
+```php
+static $meeting_search_radius = 30;
+```
+This would set the radius to a maximum of 30 miles.
+
+```php
+static $meeting_search_radius = -50;
+```
+This would set the radius to a integer of -50 and is the default.
```
Comment (radius314) [inline README.md]: If the number is negative it means the first n results.  In this case the first 50 regardless of distance.

**Pull request merged**